grafana-plugin/src/utils/decorators.ts

```diff
@@ -34,6 +34,23 @@ type GlobalNotificationConfig = {
   failureType?: 'error' | 'warning';
 };
 
+export function WrapWithGlobalNotification(
```

Comment:
As mentioned before, let's consider doing it only in the store layer, then maybe we don't need this util